test/Microsoft.ComponentDetection.Orchestrator.Tests/Services/DetectorProcessingServiceTests.cs

```diff
@@ -450,7 +450,7 @@ public void ProcessDetectorsAsync_CapturesTelemetry()
             });
 
             records.Should().Contain(x => x is DetectorExecutionTelemetryRecord);
-            records.Count(x => x is DetectorExecutionTelemetryRecord)
```

Comment:
I just want to double check these are equivalent? I am not super familiar with pattern matching syntax.

Reply:
the idea is that `records` is a collection of Type `DetectorExecutionTelemetryRecord` so, it can't be anything other than  `null` or that type.